content/app/github/wrappers/line-element-wrapper.js

```diff
@@ -11,9 +11,46 @@ define("content/app/github/wrappers/line-element-wrapper", function(require, exp
 		$statics : {
```

Comment:
I think this approach is not elegant. Basically, you are trying to handle standard lines and expandable ones in the same class, although they are different ones. I especially do not like the way we store the same value twice in _extractLineRange method.

Could we have an abstract class and two classes derived from it for handling line numbers? For example we could have AbstracLinePosition, and then extend from it StandardLinePosition and ExpandableLinePosition. The only interface (for now) would be **containsLine** method. We could event export all of them from one module, since they would be really simple.